packages/unit-testing/README.md

```diff
@@ -13,8 +13,8 @@ Suitecloud Unit Testing allows you to use unit testing with [Jest](https://jestj
 ## Features
 - Provides a default configuration to run unit tests with Jest in SuiteCloud projects.
 - Supports unit testing for SuiteScript 2.x files.
-- Provides SuiteScript 2.x stubs for SuiteScript modules.
-- Allows you to create custom stubs for SuiteScript 2.x or custom modules.
+- Provides SuiteScript 2.x stubs for all SuiteScript modules.
+- Allows you to create custom stubs for any SuiteScript 2.x or custom modules.
```

Comment:
We could change it for:
- Provides stubs for all SuiteScript 2.x modules.
- Allows you to create custom stubs for any module used in SuiteScript 2.x files.